src/Tesa/tests/phpunit/Unit/SanitizerTest.php

```diff
@@ -76,12 +76,6 @@ public function testReduceLengthToNearestWholeWordForLatinString() {
 	}
 
 	public function testReduceLengthToNearestWholeWordForNonLatinString() {
-		if ( version_compare( phpversion(), '5.4', '<' ) ) {
```

Comment:
5.4 :scream:

Reply:
There was the `smwShowFactbox` hook from SMW 1.11 and some MW 1.1X era code too haha 😱